octobot_evaluators/evaluator/TA_evaluator.py

```diff
@@ -26,14 +29,6 @@ def __init__(self):
         self.short_term_averages = [7, 5, 4, 3, 2, 1]  # TODO remove
         self.long_term_averages = [40, 30, 20, 15, 10]  # TODO remove
```

Comment:
Do we still use these values ?

Reply:
in RSIMomentumEvaluator, I move it there